src/sqlfluff/rules/L047.py

```diff
@@ -36,17 +36,18 @@ class Rule_L047(BaseRule):
 
     .. _ANSI-92: http://msdn.microsoft.com/en-us/library/ms175997.aspx
 
-    | **Anti-pattern**
+    **Anti-pattern**
 
     .. code-block:: sql
 
         select
             count(1)
         from table_a
 
-    | **Best practice**
-    | Use ``count(*)`` unless specified otherwise by config ``prefer_count_1``,
-    | or ``prefer_count_0`` as preferred.
+    **Best practice**
+
+    Use ``count(*)`` unless specified otherwise by config ``prefer_count_1``,
+    or ``prefer_count_0`` as preferred.
```

Comment:
```suggestion
    Use the configured style for "count number of rows", ``count(*)``, `count(1)``, or ``count(0)``.
```

Reply:
I actually think the original is clearer. If this was one copy (e.g. `count_style="*"`) I'd agree with your wording but, to preserve backwards compatibility, it was implemented in a bit of an awkward way (I remember cause I did this!).